setup/mail-postfix.sh

```diff
@@ -200,7 +200,7 @@ tools/editconf.py /etc/postfix/main.cf virtual_transport=lmtp:[127.0.0.1]:10025
 # "450 4.7.1 Client host rejected: Service unavailable". This is a retry code, so the mail doesn't properly bounce. #NODOC
 tools/editconf.py /etc/postfix/main.cf \
 	smtpd_sender_restrictions="reject_non_fqdn_sender,reject_unknown_sender_domain,reject_authenticated_sender_login_mismatch,reject_rhsbl_sender dbl.spamhaus.org" \
-	smtpd_recipient_restrictions=permit_sasl_authenticated,permit_mynetworks,"reject_rbl_client zen.spamhaus.org",reject_unlisted_recipient,"check_policy_service inet:127.0.0.1:10023"
+	smtpd_recipient_restrictions=permit_sasl_authenticated,permit_mynetworks,"reject_rbl_client zen.spamhaus.org","reject_rbl_client multi.uribl.com","reject_rbl_client bl.spamcop.net","reject_rbl_client cbl.abuseat.org",reject_unlisted_recipient,"check_policy_service inet:127.0.0.1:10023"
```

Comment:
Style-wise, @tdulcet's work using ShellCheck suggests quotes around the entire line - which makes sense as the entire line is just a bash string. See https://github.com/mail-in-a-box/mailinabox/pull/1457/files#diff-2c91206d3285034a200b23d8c296b8e5R203

Reply:
True, I just kept the existing style.  Will be up to Josh on what way it ends up.  Not sure if one way or the other is better git blame wise.